### PR TITLE
ENG-857 Change add button position in category and groups pages

### PR DESCRIPTION
**Pull request by raxkaynan**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **brenoqvds**

If they are not already assigned, you can assign the PR to them by writing `/assign @brenoqvds` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-admin-console).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-admin-console/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/main/webapp/WEB-INF/apsadmin/jsp/user/group/groupFinding.jsp [hunk 1]
```diff
@@ -114,3 +105,12 @@
...
 </s:else>
+<br>
+<div class="row form-group">
+    <div class="col-sm-12">
+        <a class="btn btn-primary pull-right"
+           href="<s:url namespace="/do/Group" action="new" />" style="margin-bottom: -15px"> <s:text
+                name="title.groupManagement.groupNew" />
+        </a>
+    </div>
+</div>
```
Comment (ichalagashvili) [inline src/main/webapp/WEB-INF/apsadmin/jsp/user/group/groupFinding.jsp:116]: new line please

Comment (raxkaynan) [reply src/main/webapp/WEB-INF/apsadmin/jsp/user/group/groupFinding.jsp]: good catch

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-admin-console/pull/21#pullrequestreview-452440727" title="Approved">gidesan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-admin-console).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/entando-admin-console/blob/master/OWNERS)~~ [gidesan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**